lib/config/yml_generator_config.dart

```diff
@@ -342,4 +362,32 @@ class YmlGeneratorConfig {
           'Could not generate all models. `$type` is not added to the config file, but is extended. These types are known: ${names.join(',')}');
     }
   }
+
+  ItemType _parseSimpleType(String type) {
+    final listRegex = RegExp(r'List<\s*([^<>]*)\s*>');
+    final mapRegex = RegExp(r'Map<([^<>,]*)\s*,\s*([^<>,]*)\s*>');
+
+    if (type == 'object' || type == 'dynamic' || type == 'any') {
+      return DynamicType();
+    } else if (type == 'bool' || type == 'boolean') {
+      return BooleanType();
```

Comment:
What we could do, is have static methods on the types that do the matching instead

Reply:
Will create a ticket for this: https://github.com/icapps/flutter-model-generator/issues/116